rootfs/templates/wrapper-body.sh

```diff
@@ -0,0 +1,530 @@
+# Default directory mounts for the user's home directory
```

Comment:
1. No, this is a template fragment, the shebang is provided in an earlier template
2. `homedir_default_mounts` has to be a string because it needs to be exported to a subshell